workflows/release-helm-charts.yml

```diff
@@ -0,0 +1,109 @@
+---
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Intel Corporation
+# Copyright 2024 Kyunghee University
+# Copyright 2024 The Linux Foundation
+
+name: Publish image and tag/release code
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  version-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: check version
+        run: |
+          sudo snap install yq
```

Comment:
Is not there already a GHA for `yq`? Also, any reason to install it using snap instead of using its source?

Reply:
This was based on [sdran-helm-charts' release action](https://github.com/onosproject/sdran-helm-charts/blob/master/.github/workflows/release.yml), but you're completely correct. I've removed the snap installations and replaced them with an action.